### PR TITLE
Add "not_present" validation

### PR DESCRIPTION
**Pull request by doha99**

### CODE REVIEW

File: src/Rules/NotPresent.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    /**
+     * Check whether attribute value is set or not.
+     *
+     * @param  string  $attribute Attribute name
+     * @param  mixed  $value Attribute value
+     * @return boolean
+     */
+    public function passes($attribute, $value)
+    {
+        return is_null($value);
```
Comment (VladimirBerdnik) [inline src/Rules/NotPresent.php]: Rule described as 'attribute not present' but checked as 'attribute is empty'. According to existing laravel's 'present' rule we should check that attribute should not be presented in array of validated attributes. We shoud change name of this rule to 'empty' or change code to check that attribute not presented in array of validated attributes.

File: tests/NotPresentRuleTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        $this->assertEquals('not_present', Rule::notPresent());
+    }
+
+    /**
+     * Test "not_present" rule with mixed usage
+     */
+    public function testMixedForNotPresent()
+    {
+        $rules = Rule::string()->notPresent()->required()->min(3);
+        $this->assertEquals('string|not_present|required|min:3', $rules);
```
Comment (VladimirBerdnik) [inline tests/NotPresentRuleTest.php]: I know, that this rule is for test purposes only, but how it is possible that attribute should not be presented and should be required?

File: tests/NotPresentValidatorTest.php [hunk 3]
```diff
@@ -57,6 +57,14 @@ public function testNotPresentInvalid()
         $this->assertFalse($result);
     }
 
+    public function testNotPresentInvalid2()
```
Comment (doha99) [inline tests/NotPresentValidatorTest.php]: @VladimirBerdnik Please take a look at this test function.

File: src/Rules/NotPresent.php (hunk 1 above)
Comment (doha99) [reply src/Rules/NotPresent.php]: As I know, If the field is not present, the value is always null. 
I added a function to test this case and mentioned you.

Comment (VladimirBerdnik) [reply src/Rules/NotPresent.php]: Let's imagine that we want to take all parameters from request and update model according to this parameters.

For example we write rule for field 'age' as 'not_present', but it is presented (but null) and after updating model by all request fields age will be updated to null

File: tests/NotPresentValidatorTest.php [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        ], [
+            'field' => 'not_present',
+        ])->passes();
+        $this->assertFalse($result);
+    }
+
+    public function testNotPresentValidWithFluentWay()
+    {
+        $result = $this->validator->make([
+            'field' => null,
```
Comment (VladimirBerdnik) [inline tests/NotPresentValidatorTest.php]: Field 'field' is presented in arry of fields but test passed. This is unexpected behaviour and differs from rule description. Now rule works not as 'not_present' but like 'should_be_empty'

Comment (doha99) [reply tests/NotPresentValidatorTest.php]: I have just committed new code. It fixed this case.

**Pull request closed without merging**